ktx/src/main/java/co/infinum/goldfinger/ktx/GoldfingerKtx.kt

```diff
@@ -0,0 +1,75 @@
+package co.infinum.goldfinger.ktx
+
+import android.content.Context
+import co.infinum.goldfinger.Goldfinger
+import co.infinum.goldfinger.Goldfinger.PromptParams
+import co.infinum.goldfinger.crypto.CipherCrypter
+import co.infinum.goldfinger.crypto.CipherFactory
+import co.infinum.goldfinger.crypto.MacCrypter
+import co.infinum.goldfinger.crypto.MacFactory
+import co.infinum.goldfinger.crypto.SignatureCrypter
+import co.infinum.goldfinger.crypto.SignatureFactory
+import kotlinx.coroutines.flow.Flow
+
+interface GoldfingerKtx {
+
+    fun hasFingerprintHardware(): Boolean
+
+    fun hasEnrolledFingerprint(): Boolean
+
+    fun canAuthenticate(): Boolean
+
+    suspend fun authenticate(
+        params: PromptParams,
+    ): Flow<Goldfinger.Result>
+
+    suspend fun encrypt(
+        params: PromptParams,
+        key: String,
+        value: String,
+    ): Flow<Goldfinger.Result>
+
+    suspend fun decrypt(
+        params: PromptParams,
+        key: String,
+        value: String,
+    ): Flow<Goldfinger.Result>
```

Comment:
The functions returning `Flow` should not be suspending, which you can do easily since in the implementation `inGoldfingerFlow` is not suspending

Reply:
Indeed, will correct